hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/DataPolicyNode.tsx

```diff
@@ -17,38 +17,38 @@ export const DataPolicyNode: FC<NodeProps<DataPolicyData>> = (props) => {
       <NodeWrapper route={`node/${DataHubNodeType.DATA_POLICY}/${id}`} {...props}>
         <HStack>
           <NodeIcon type={DataHubNodeType.DATA_POLICY} />
-          <Text> {t('workspace.nodes.type', { context: type })}</Text>
+          <Text data-testid={'node-title'}> {t('workspace.nodes.type', { context: type })}</Text>
```

Comment:
suggestion: let's assign the attribute without the evaluation

```suggestion
          <Text data-testid="node-title"> {t('workspace.nodes.type', { context: type })}</Text>
```